autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -70,10 +78,9 @@ const CourseSelectColumn: React.FC = () => {
       });
     };
 
-    throttle(`saveCourseCards${term}`, saveCourses, 15000, true);
+    throttle(`${term}`, saveCourses, 15000, true);
```

Comment:
What's your reasoning for removing the `saveCourseCards` in the key? Are there no other functions that use `throttle`?

Reply:
Wait, so I read the rest of the code, and I think what happened is that your nre `createThrottleFunction` gives a separate state to each file that imports and calls it. Therefore, if you used this function in a new file, it would have a different state, so there would be no possibility for conflict with this usage here. Did I understand that correctly?

Reply:
Yeah that's correct